airbyte-integrations/connectors/source-bing-ads/unit_tests/integrations/base_test.py

```diff
@@ -21,6 +21,15 @@
 
 
 class BaseTest(TestCase):
+    def setUp(self) -> None:
```

Comment:
Using `@HttpMocker()` as an annotation is not very good with inheritance because it is only instantiated once per method regardless of the number of instances. This was fine until [this](https://github.com/airbytehq/airbyte-python-cdk/commit/4f68840e1dd6ae3ec928f936b25b216f791af693#diff-e32a41a59b609a26cd6a3ff5b306eef6a44c1948b82e01ae287c3377ff1fb5ebR60-R61) was added. In order to fix this, I make http_mocker an instance variable